src/Arcus.Templates.ServiceBus.Queue/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Arcus.Security.Core;
+using Arcus.Security.Core.Caching;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Arcus.Templates.ServiceBus.Queue
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args)
+                .Build()
+                .Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(configuration =>
+                {
+                    configuration.AddCommandLine(args);
+                    configuration.AddEnvironmentVariables();
+                })
+                .ConfigureServices((hostContext, services) =>
+                {
+                    //#error Please provide a valid secret provider, for example Azure Key Vault: https: //security.arcus-azure.net/features/secrets/consume-from-key-vault
+                    services.AddSingleton<ISecretProvider>(serviceProvider => new CachedSecretProvider(secretProvider: null));
+
+                    services.AddServiceBusQueueMessagePump<EmptyMessagePump>(secretProvider => secretProvider.GetRawSecretAsync("ARCUS_SERVICEBUS_CONNECTIONSTRING"));
+                    services.AddTcpHealthProbes();
```

Comment:
Should I a single sample health check + alter the deserialization in the tests (according to [this issue](https://github.com/arcus-azure/arcus.messaging/issues/21))

Reply:
I think this is ok, all the infrastructure is there so it's up to them to extend it, or not?

Both options are ok for me!

Reply:
Ok, I'll resolve this.